### DISCUSSION TITLE
Parent doesn't listen the gestures from its child

### DISCUSSION THREAD

**Discussion by ShaliniSF79** [dotnet/maui#21278] [Ideas]
### Description

**Issue:**
I have a simple sample in .NET MAUI using Grid and Label controls. I've added tap gestures to both controls to listen for tap events. Currently, when I tap the grid control, its tap event is invoked, and the same for the label. For my requirement, I need to trigger the grid's(Parent) tap event when tapping the label's(Child) tap gesture. This doesn't work by default. It seems like behaviour. Is there a way to achieve this?

### Steps to Reproduce

1. Run the given sample.
2. Tap `Tap me` button.
3. You can see the tap event listens only for the label.

### Link to public reproduction project repository

https://github.com/ShaliniSF79/MauiDemo

### Version with bug

8.0.7 SR2

### Is this a regression from previous behavior?

No, this is something new

### Last version that worked well

Unknown/Other

### Affected platforms

Android, Windows

### Did you find any workaround?

No

**Comment by PureWeen** [comment]
@ShaliniSF79 do you want both gesture recognizers to fire when you tap the label?

**Comment by ShaliniSF79** [comment]
> @ShaliniSF79 do you want both gesture recognizers to fire when you tap the label?

Yes @PureWeen, I'm expecting, when I tap the label, both grid and label gestures have to be fired.

**Comment by ninachen03** [comment]
Can repro this issue at windows platform on the latest VS17.10.0 preview2.
![Tapme](https://github.com/dotnet/maui/assets/128687574/05395a6a-aaf0-4d2c-b8d3-9e8892d297a4)

**Comment by PureWeen** [comment]
Currently this behavior isn't supported. I've moved it to an idea to see if it's something we'd want to add. 

You could just fire on the grid and use the coordinates to see if it's clicking the label
or
propagate it yourself from the label up to the grid.
